AirCasting/Notes/NotesHandler.swift

```diff
@@ -44,13 +44,15 @@ class NotesHandlerDefault: NSObject, NotesHandler, NSFetchedResultsControllerDel
     }
     
     func addNote(noteText: String) {
+        let latitude = locationTracker.locationManager.location?.coordinate.latitude ?? locationTracker.googleLocation.last?.location.latitude ?? 20.0
```

Comment:
I add google location because Janek told me about the case when the user turns off the location in settings when the mobile session is active, it's not a rare case but it covers it. These three values (2 default) cover all cases 1st session with location, the second when the user turns off the location in settings when the mobile session is active, and the third in location less session when it doesn't matter where the note will be placed. 

Also, I removed withlocation parameter but I don't know why it is included in this commit ...

I hope everything will be clear if not please let me know.

Reply:
I think we would want to use this `withLocation` parameter, because when user turns on `disable location` is settings, the location manager still has it's location, so we are saving a real location instead of the fake one. At least that's what I found with those logs 

<img width="1021" alt="Screenshot 2022-04-14 at 15 40 00" src="https://user-images.githubusercontent.com/15946812/163402880-9c95f873-eb76-43fe-b235-bb75a55c148d.png">

Reply:
Maybe I do not understand it correctly but I checked which value is used and when the user turns off location during the recording session the location is used from the code `locationTracker.googleLocation.last?.location.longitude`. When I removed the above code the location was used with default values.

I removed the "with location" property because is not used when we fetch localization only in NotesHandler.